client/ayon_core/pipeline/publish/lib.py

```diff
@@ -694,67 +639,85 @@ def get_custom_staging_dir_info(
     product_name,
     project_settings=None,
     anatomy=None,
-    log=None
+    log=None,
 ):
-    """Checks profiles if context should use special custom dir as staging.
+    from ayon_core.pipeline.stagingdir import get_staging_dir_config
+
```

Comment:
Requires `import warnings`.
```suggestion
    warnings.warn(
        (
            "Function 'get_custom_staging_dir_info' in"
            " 'ayon_core.pipeline.publish' is deprecated. Please use"
            " 'get_custom_staging_dir_info'"
            " in 'ayon_core.pipeline.stagingdir'."
        ),
        DeprecationWarning,
    )
```